terraform-module/main.tf

```diff
@@ -53,6 +53,7 @@ module "cdn" {
   domain_name   = local.domain_name
   bucket_name   = module.s3.bucket_name
   block_iframes = var.block_iframes
+  is_indexed    = var.is_indexed
```

Comment:
Would it make sense to add the word robots in there? Indexed can mean a lot of things

Reply:
true, renamed this to `is_robots_indexing_allowed`. a bit long but i think it should be easy to understand.